typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.stories.tsx

```diff
@@ -415,6 +454,122 @@ AllWellHeadsHidden.parameters = {
     },
 };
 
+const testWell: FeatureCollection = {
+    type: "FeatureCollection",
+    features: [
+        {
+            type: "Feature",
+            geometry: {
+                type: "GeometryCollection",
+                geometries: [
+                    {
+                        type: "Point",
+                        coordinates: [0, 0],
+                    },
+                    {
+                        type: "LineString",
+                        coordinates: [
+                            [0, 0, 0],
+                            [0, 0, 1],
+                            [0, 0, 2],
+                            [0, 50, -50],
+                            [0, 0, -100],
+                            [99, 99, -150],
+                            [99, 0, -250],
+                        ],
+                    },
+                ],
+            },
+            properties: {
+                name: "well99",
+                color: [255, 255, 0, 255],
+                md: [[0, 1, 2, 3, 4, 5, 8, 9]],
+            },
+        },
+    ],
+};
+
+const BBox = [-100, -100, -250, 100, 100, 0] as [
+    number,
+    number,
+    number,
+    number,
+    number,
+    number,
+];
+
+export const WellsRefine: StoryFn<typeof SubsurfaceViewer> = (args) => {
```

Comment:
I can't see the story, is it just me?

![image](https://github.com/equinor/webviz-subsurface-components/assets/19621822/6a51c0fc-444d-4009-a784-3a42568e3a65)

Reply:
It should be there.